ui/pages/confirmations/components/confirm/title/title.tsx

```diff
@@ -132,14 +142,37 @@ const ConfirmTitle: React.FC = memo(() => {
 
   const { isNFT } = useIsNFT(currentConfirmation as TransactionMeta);
 
+  let isRevokeSetApprovalForAll = false;
+  if (
+    currentConfirmation?.type === TransactionType.tokenMethodSetApprovalForAll
+  ) {
+    const decodedResponse = useDecodedTransactionData();
+
+    isRevokeSetApprovalForAll =
+      decodedResponse.value?.data[0].name === 'setApprovalForAll' &&
```

Comment:
Since we have this same logic elsewhere, is it worth a small `useIsApprovalForAll` hook?

Reply:
Decoded response is used on the other occurrence, so I can't extract it to the hook. I extracted the duplicated logic to a util function instead.